data_managers/data_manager_fetch_busco/data_manager/data_manager.py

```diff
@@ -1,71 +1,47 @@
+#!/usr/bin/env python
+#
+# Data manager for reference data for the 'BUSCO' Galaxy tools
 import argparse
 import datetime
 import json
 import os
 import shutil
-import tarfile
-import zipfile
-try:
-    # For Python 3.0 and later
-    from urllib.request import Request, urlopen
-except ImportError:
-    # Fall back to Python 2 imports
-    from urllib2 import Request, urlopen
-
-
-def url_download(url, workdir):
-    file_path = os.path.join(workdir, 'download.dat')
-    if not os.path.exists(workdir):
-        os.makedirs(workdir)
-    src = None
-    dst = None
-    try:
-        req = Request(url)
-        src = urlopen(req)
-        with open(file_path, 'wb') as dst:
-            while True:
-                chunk = src.read(2**10)
-                if chunk:
-                    dst.write(chunk)
-                else:
-                    break
-    finally:
-        if src:
-            src.close()
-    if tarfile.is_tarfile(file_path):
-        fh = tarfile.open(file_path, 'r:*')
-    elif zipfile.is_zipfile(file_path):
-        fh = zipfile.ZipFile(file_path, 'r')
-    else:
-        return
-    fh.extractall(workdir)
-    os.remove(file_path)
+import subprocess
+from pathlib import Path
 
 
 def main(args):
-    workdir = os.path.join(os.getcwd(), 'busco')
-    url_download(args.url, workdir)
+    workdir = os.path.join(os.getcwd(), "busco_downloads")
+    cmd = "busco --download %s" % args.database
+    subprocess.check_call(cmd, shell=True)
+    with open(args.json) as fh:
+        params = json.load(fh)
+    target_directory = params["output_data"][0]["extra_files_path"]
     data_manager_entry = {}
-    data_manager_entry['value'] = args.name.lower()
-    data_manager_entry['name'] = args.name
-    data_manager_entry['path'] = '.'
+    data_manager_entry["value"] = args.name.lower()
+    data_manager_entry["name"] = args.name
+    data_manager_entry["version"] = args.version
+    data_manager_entry["path"] = str(Path(target_directory))
     data_manager_json = dict(data_tables=dict(busco=data_manager_entry))
```

Comment:
I think you need to adapt the data table name here too:
```suggestion
    data_manager_json = dict(data_tables=dict(busco_database=data_manager_entry))
```

Reply:
Perfect, thanks a lot!